translations/README.es.txt

```diff
@@ -0,0 +1,116 @@
+• OpenEBS
+
+[! [Liberaciones] (https://img.shields.io/github/release/openebs/openebs/all.svg?style=flat-square)] (https://github.com/openebs/openebs/releases)
+[! [#openebs canal de Slack] (https://img.shields.io/badge/slack-openebs-brightgreen.svg?logo=slack)] (https://kubernetes.slack.com/messages/openebs)
+[! [Twitter] (https://img.shields.io/twitter/follow/openebs.svg?style=social&label=Follow)] (https://twitter.com/intent/follow?screen_name=openebs)
+[! [Bienvenidos a los PRP] (https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)] (https://github.com/openebs/openebs/blob/master/CONTRIBUTING.md)
+[! [Estado FOSSA] (https://app.fossa.com/api/projects/git%2Bgithub.com%2Fopenebs%2Fopenebs.svg?type=shield)] (https://app.fossa.com/projects/git%2Bgithub.com%2Fopenebs%2Fopenebs?ref=badge_shield)
+[! [Prácticas recomendadas de CII] (https://bestpractices.coreinfrastructure.org/projects/1754/badge)] (https://bestpractices.coreinfrastructure.org/projects/1754)
+
+https://openebs.io/
+

```

Comment:
Please check that there is a change in how the other translations are listed.

Reply:
@kmova I have made the changes that you asked. Please let me know if later commits make any problem.